src/ImmersedBoundaries/immersed_grid_metrics.jl

```diff
@@ -35,6 +35,9 @@ end
 
 
 return_metrics(grid::ImmersedBoundaryGrid) = return_metrics(grid.underlying_grid)
+Δx(grid::ImmersedBoundaryGrid) = Δx(grid.underlying_grid)
+Δy(grid::ImmersedBoundaryGrid) = Δy(grid.underlying_grid)
+Δz(grid::ImmersedBoundaryGrid) = Δz(grid.underlying_grid)
```

Comment:
I don't think we should define no-location versions

Reply:
Agreed. Fixed.